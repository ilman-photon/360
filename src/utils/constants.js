module.exports = {
  EMPTY_STRING: "",
  H1: "h1",
  H2: "h2",
  SMALL: "small",
  LARGE: "large",
  PATIENT: "patient",
  PRIMARY: "primary",
  SECONDARY: "secondary",
  MIDDLE: "middle",
  CENTER: "center",
  FLEX_END: "flex-end",
  BODY_REGULAR: "bodyRegular",
  FILLED: "filled",
  INPUT_TEXT: "text",
  INPUT_PASSWORD: "password",
  ACCOUNT_LOCK_COUNT: 3,
  SUBMIT: "submit",
  SELECT_OPTION: "selectOption",
  SECURITY_QUESTION: "securityQuestion",
  ONE_TIME_LINK: "oneTimeLink",
  PASSWORD_RESET: "passwordReset",
  BOTH: "both",
  MODE_COMMUNICATION_KEY: "modeComunication",
  EMAIL: "email",
  PHONE: "phone",
  YES: "yes",
  MFA_COMPONENT_NAME: "MultiFactorAuthentication",
  SQ_COMPONENT_NAME: "SecurityQuestion",
  MONTH_NAME: [
    "Jan",
    "Feb",
    "Mar",
    "Apr",
    "May",
    "Jun",
    "Jul",
    "Aug",
    "Sep",
    "Oct",
    "Nov",
    "Dec",
  ],
  DAY_NAME: ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat"],
  TEST_ID: {
    LOGIN_TEST_ID: {
      email: "emailorphonenumber",
      password: "password",
      forgotLink: "forgotpswd",
      loginBtn: "loginbtn",
      guestBtn: "continueasguest",
      createAccountBtn: "createaccount",
      syncAppointmentLink: "syncAppointmentLink",
    },
    FORGOT_TEST_ID: {
      email: "emailorphonenumber",
      continueBtn: "continuebtn",
      loginLink: "backtologin",
      answerQuestions: "answerquestions",
      oneTimeLink: "onetimelink",
      securityQuestion: "securityquestion",
    },
    REGISTER_TEST_ID: {
      firstname: "firstname",
      lastname: "lastname",
      dateofbirth: "dateofbirth",
      calander: "calander",
      email: "email",
      mobilenumber: "mobilenumber",
      password: "password",
      emailradio: "emailradio",
      phoneradio: "phoneradio",
      bothradio: "bothradio",
      registerbtn: "registerBtn",
      loginlink: "loginlink",
    },
    SCHEDULE_GUEST_TEST_ID: {
      firstname: "firstname",
      lastname: "lastname",
      dateofbirth: "dateofbirth",
      calander: "calander",
      email: "email",
      mobilenumber: "mobilenumber",
      password: "password",
      emailradio: "emailradio",
      phoneradio: "phoneradio",
      bothradio: "bothradio",
      registerbtn: "registerBtn",
      signInlink: "signInlink",
    },
    HOME_TEST_ID: {
      appoinments: "appoinments",
      myhealthchart: "myhealthchart",
      mycareteam: "mycareteam",
      labresults: "labresults",
      billing: "billing",
      username: "username",
      account: "account",
      logout: "logout",
    },
    MFA_TEST_ID: {
      radioEmail: "loc_RadioEmail",
      radioPhone: "loc_RadioPhone",
      rememberMe: "loc_chkRemember",
      btnConfirm: "loc_btnConfirm",
      btnBack: "loc_btnBack",
      inputCode: "loc_txtEnterCode",
      btnSubmit: "loc_btnSubmit",
      btnResend: "loc_btnResend",
      btnSubmitSecurity: "loc_btnSubmitSecurity",
      btnSkip: "loc_btnSkip",
    },
    PERSONAL_INFO_TEST_ID: {
      txtName: "loc_txtName",
      btnSave: "loc_btnSave",
      btnCancel: "loc_btnCancel",
      btnDelete: "loc_btnDelete",
    },
    APPOINTMENT_TEST_ID: {
      location: "location",
      searchbtn: "searchbtn",
      filterbtn: "filterbtn",
      locationInput: "locationInput",
      dateInput: "dateInput",
      purposeInput: "purposeInput",
      insuranceInput: "insuranceInput",
      FILTER_RESULT: {
        container: "appointment_filter_result_container",
      },
      FILTER_COMPONENT: {
        drawer: "appointment_filter_component_drawer",
      },
    },
    BIOGRAPHY_TEST_ID: {
      about: "biography-about",
      location: "biography-location",
      insurance: "biography-insurance",
      education: "biography-education",
      viewAll: "biography-view-all-insurance",
    },
    INSURANCE_TEST_ID: {
      addButton: "loc_btnAddInsurance",
      provider: "loc_inputProvider",
      planName: "loc_inputPlan",
      subscriberId: "loc_inputSubsId",
      group: "loc_inputGroup",
      save: "loc_btnSaveInsuranve",
      cancel: "loc_btnCancelInsuranve",
      uploadFrontImage: "loc_btnUploadFront",
      uploadBackImage: "loc_btnUploadBack",
    },
    SEARCH_PROVIDER_TEST_ID: {
      viewAll: "loc_viewAll",
    },
    SUBNAVIGATION: "subNavigation-link",
    SCHEDULE_APPOINTMENT_TEST_ID: {
      APPOINTMENT_LOCATION: {
        editButton: "schedule_appointment_location_edit_button",
      },
      APPOINTMENT_DETAILS: {
        editButton: "schedule_appointment_details_edit_button",
      },
      step2Button: "schedule_appointment_step2_button",
    },
  },
};
