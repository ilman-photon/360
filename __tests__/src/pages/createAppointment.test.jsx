import { useMediaQuery } from "@mui/material";
import "@testing-library/jest-dom";
import { cleanup, render, waitFor } from "@testing-library/react";
import CreateAppointmentPage from "../../../src/pages/patient/appointments/create";
import constants from "../../../src/utils/constants";

describe("Render Create Appointment", () => {
  let container;
  const TEST_ID = constants.TEST_ID.APPOINTMENT_TEST_ID;
  beforeEach(() => {
    container = render(<CreateAppointmentPage />);
  });

  test("is Create Appointment page render", async () => {
    await waitFor(() => {
      container.getByTestId("LocationOnOutlinedIcon");
    });
    expect(container.getByTestId("LocationOnOutlinedIcon")).toBeInTheDocument();
  });
});
