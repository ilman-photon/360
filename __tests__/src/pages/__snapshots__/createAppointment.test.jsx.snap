// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Create Appointment is Create Appointment page render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root css-13pmxen"
      >
        <div
          class="mobileFilterContainer MuiBox-root css-0"
        >
          <div
            class="mobileContainer MuiBox-root css-0"
          >
            <h2
              class="MuiTypography-root MuiTypography-h2 mobileTitle css-1sra7t5-MuiTypography-root"
            >
              Schedule an eye exam
            </h2>
            <form>
              <div
                class="MuiBox-root css-0"
              >
                <div
                  class="MuiBox-root css-1iqw0sw"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-145921m-MuiSvgIcon-root"
                    data-testid="LocationOnOutlinedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zM7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 2.88-2.88 7.19-5 9.88C9.92 16.21 7 11.85 7 9z"
                    />
                    <circle
                      cx="12"
                      cy="9"
                      r="2.5"
                    />
                  </svg>
                  <div
                    class="MuiFormControl-root MuiTextField-root custom-input css-11mrpan-MuiFormControl-root-MuiTextField-root"
                    withicon="true"
                  >
                    <label
                      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled MuiFormLabel-root MuiFormLabel-colorPrimary css-1suhfmo-MuiFormLabel-root-MuiInputLabel-root"
                      data-shrink="false"
                      for=":r0:"
                      id=":r0:-label"
                    >
                      City, state, or zip code
                    </label>
                    <div
                      class="MuiFilledInput-root MuiFilledInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-adornedEnd css-1r1rxmt-MuiInputBase-root-MuiFilledInput-root"
                    >
                      <input
                        aria-invalid="false"
                        class="MuiFilledInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-1gctnaj-MuiInputBase-input-MuiFilledInput-input"
                        id=":r0:"
                        placeholder=""
                        readonly=""
                        type="default"
                        value=""
                      />
                      <div
                        class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-filled MuiInputAdornment-sizeMedium css-jv61v3-MuiInputAdornment-root"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-q52vwa-MuiSvgIcon-root"
                          data-testid="NearMeOutlinedIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="m17.27 6.73-4.24 10.13-1.32-3.42-.32-.83-.82-.32-3.43-1.33 10.13-4.23M21 3 3 10.53v.98l6.84 2.65L12.48 21h.98L21 3z"
                          />
                        </svg>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-0"
              >
                <div
                  class="MuiBox-root css-1iqw0sw"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-13cmedr-MuiSvgIcon-root"
                    data-testid="CalendarTodayIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"
                    />
                  </svg>
                  <div
                    class="MuiFormControl-root MuiTextField-root custom-input css-11mrpan-MuiFormControl-root-MuiTextField-root"
                    withicon="true"
                  >
                    <label
                      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-filled MuiFormLabel-root MuiFormLabel-colorPrimary MuiFormLabel-filled css-11l7838-MuiFormLabel-root-MuiInputLabel-root"
                      data-shrink="true"
                      for=":r1:"
                      id=":r1:-label"
                    >
                      Date
                    </label>
                    <div
                      class="MuiFilledInput-root MuiFilledInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-qax0ed-MuiInputBase-root-MuiFilledInput-root"
                    >
                      <input
                        aria-invalid="false"
                        class="MuiFilledInput-input MuiInputBase-input css-10botns-MuiInputBase-input-MuiFilledInput-input"
                        id=":r1:"
                        placeholder=""
                        readonly=""
                        type="default"
                        value="Sep 2, 2022"
                      />
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-0"
              >
                <div
                  class="MuiBox-root css-1iqw0sw"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-127ftl4-MuiSvgIcon-root"
                    data-testid="VisibilityOutlinedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 6c3.79 0 7.17 2.13 8.82 5.5C19.17 14.87 15.79 17 12 17s-7.17-2.13-8.82-5.5C4.83 8.13 8.21 6 12 6m0-2C7 4 2.73 7.11 1 11.5 2.73 15.89 7 19 12 19s9.27-3.11 11-7.5C21.27 7.11 17 4 12 4zm0 5c1.38 0 2.5 1.12 2.5 2.5S13.38 14 12 14s-2.5-1.12-2.5-2.5S10.62 9 12 9m0-2c-2.48 0-4.5 2.02-4.5 4.5S9.52 16 12 16s4.5-2.02 4.5-4.5S14.48 7 12 7z"
                    />
                  </svg>
                  <div
                    class="MuiFormControl-root MuiTextField-root custom-input css-11mrpan-MuiFormControl-root-MuiTextField-root"
                    withicon="true"
                  >
                    <label
                      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled MuiFormLabel-root MuiFormLabel-colorPrimary css-1suhfmo-MuiFormLabel-root-MuiInputLabel-root"
                      data-shrink="false"
                      for=":r2:"
                      id=":r2:-label"
                    >
                      Purposes of Visit
                    </label>
                    <div
                      class="MuiFilledInput-root MuiFilledInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-qax0ed-MuiInputBase-root-MuiFilledInput-root"
                    >
                      <input
                        aria-invalid="false"
                        class="MuiFilledInput-input MuiInputBase-input css-10botns-MuiInputBase-input-MuiFilledInput-input"
                        id=":r2:"
                        placeholder=""
                        readonly=""
                        type="default"
                        value=""
                      />
                    </div>
                  </div>
                </div>
                <span
                  class="MuiTypography-root MuiTypography-bodyTinyMedium css-lj1nv7-MuiTypography-root"
                >
                  (Optional)
                </span>
              </div>
              <div
                class="MuiBox-root css-0"
              >
                <div
                  class="MuiBox-root css-1iqw0sw"
                >
                  <div
                    class="MuiBox-root css-132diu7"
                  >
                    <span
                      style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
                    >
                      <span
                        style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                      >
                        <img
                          alt=""
                          aria-hidden="true"
                          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2720%27%20height=%2720%27/%3e"
                          style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                        />
                      </span>
                      <img
                        alt=""
                        data-nimg="intrinsic"
                        decoding="async"
                        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                        style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                      />
                      <noscript />
                    </span>
                  </div>
                  <div
                    class="MuiFormControl-root MuiTextField-root custom-input css-11mrpan-MuiFormControl-root-MuiTextField-root"
                    withicon="true"
                  >
                    <label
                      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled MuiFormLabel-root MuiFormLabel-colorPrimary css-1suhfmo-MuiFormLabel-root-MuiInputLabel-root"
                      data-shrink="false"
                      for=":r3:"
                      id=":r3:-label"
                    >
                      Insurance Carrier
                    </label>
                    <div
                      class="MuiFilledInput-root MuiFilledInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-qax0ed-MuiInputBase-root-MuiFilledInput-root"
                    >
                      <input
                        aria-invalid="false"
                        class="MuiFilledInput-input MuiInputBase-input css-10botns-MuiInputBase-input-MuiFilledInput-input"
                        id=":r3:"
                        placeholder=""
                        readonly=""
                        type="default"
                        value=""
                      />
                    </div>
                  </div>
                </div>
                <span
                  class="MuiTypography-root MuiTypography-bodyTinyMedium css-lj1nv7-MuiTypography-root"
                >
                  (Optional)
                </span>
              </div>
              <button
                class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root searchButton css-1unuvva-MuiButtonBase-root-MuiButton-root"
                data-testid="searchbtn"
                tabindex="0"
                type="submit"
              >
                Search
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </form>
          </div>
          <div />
        </div>
        <div
          class="MuiGrid-root MuiGrid-container css-u2wmcx-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1gntrgc-MuiGrid-root"
          >
            Filter Result content here
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1k4mpik-MuiGrid-root"
          >
            <span
              class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-18lrjg1-MuiCircularProgress-root"
              role="progressbar"
              style="width: 40px; height: 40px;"
            >
              <svg
                class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
                viewBox="22 22 44 44"
              >
                <circle
                  class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
                  cx="44"
                  cy="44"
                  fill="none"
                  r="20.2"
                  stroke-width="3.6"
                />
              </svg>
            </span>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root css-13pmxen"
    >
      <div
        class="mobileFilterContainer MuiBox-root css-0"
      >
        <div
          class="mobileContainer MuiBox-root css-0"
        >
          <h2
            class="MuiTypography-root MuiTypography-h2 mobileTitle css-1sra7t5-MuiTypography-root"
          >
            Schedule an eye exam
          </h2>
          <form>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiBox-root css-1iqw0sw"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-145921m-MuiSvgIcon-root"
                  data-testid="LocationOnOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zM7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 2.88-2.88 7.19-5 9.88C9.92 16.21 7 11.85 7 9z"
                  />
                  <circle
                    cx="12"
                    cy="9"
                    r="2.5"
                  />
                </svg>
                <div
                  class="MuiFormControl-root MuiTextField-root custom-input css-11mrpan-MuiFormControl-root-MuiTextField-root"
                  withicon="true"
                >
                  <label
                    class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled MuiFormLabel-root MuiFormLabel-colorPrimary css-1suhfmo-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="false"
                    for=":r0:"
                    id=":r0:-label"
                  >
                    City, state, or zip code
                  </label>
                  <div
                    class="MuiFilledInput-root MuiFilledInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-adornedEnd css-1r1rxmt-MuiInputBase-root-MuiFilledInput-root"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiFilledInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-1gctnaj-MuiInputBase-input-MuiFilledInput-input"
                      id=":r0:"
                      placeholder=""
                      readonly=""
                      type="default"
                      value=""
                    />
                    <div
                      class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-filled MuiInputAdornment-sizeMedium css-jv61v3-MuiInputAdornment-root"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-q52vwa-MuiSvgIcon-root"
                        data-testid="NearMeOutlinedIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="m17.27 6.73-4.24 10.13-1.32-3.42-.32-.83-.82-.32-3.43-1.33 10.13-4.23M21 3 3 10.53v.98l6.84 2.65L12.48 21h.98L21 3z"
                        />
                      </svg>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiBox-root css-1iqw0sw"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-13cmedr-MuiSvgIcon-root"
                  data-testid="CalendarTodayIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"
                  />
                </svg>
                <div
                  class="MuiFormControl-root MuiTextField-root custom-input css-11mrpan-MuiFormControl-root-MuiTextField-root"
                  withicon="true"
                >
                  <label
                    class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-filled MuiFormLabel-root MuiFormLabel-colorPrimary MuiFormLabel-filled css-11l7838-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="true"
                    for=":r1:"
                    id=":r1:-label"
                  >
                    Date
                  </label>
                  <div
                    class="MuiFilledInput-root MuiFilledInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-qax0ed-MuiInputBase-root-MuiFilledInput-root"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiFilledInput-input MuiInputBase-input css-10botns-MuiInputBase-input-MuiFilledInput-input"
                      id=":r1:"
                      placeholder=""
                      readonly=""
                      type="default"
                      value="Sep 2, 2022"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiBox-root css-1iqw0sw"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-127ftl4-MuiSvgIcon-root"
                  data-testid="VisibilityOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 6c3.79 0 7.17 2.13 8.82 5.5C19.17 14.87 15.79 17 12 17s-7.17-2.13-8.82-5.5C4.83 8.13 8.21 6 12 6m0-2C7 4 2.73 7.11 1 11.5 2.73 15.89 7 19 12 19s9.27-3.11 11-7.5C21.27 7.11 17 4 12 4zm0 5c1.38 0 2.5 1.12 2.5 2.5S13.38 14 12 14s-2.5-1.12-2.5-2.5S10.62 9 12 9m0-2c-2.48 0-4.5 2.02-4.5 4.5S9.52 16 12 16s4.5-2.02 4.5-4.5S14.48 7 12 7z"
                  />
                </svg>
                <div
                  class="MuiFormControl-root MuiTextField-root custom-input css-11mrpan-MuiFormControl-root-MuiTextField-root"
                  withicon="true"
                >
                  <label
                    class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled MuiFormLabel-root MuiFormLabel-colorPrimary css-1suhfmo-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="false"
                    for=":r2:"
                    id=":r2:-label"
                  >
                    Purposes of Visit
                  </label>
                  <div
                    class="MuiFilledInput-root MuiFilledInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-qax0ed-MuiInputBase-root-MuiFilledInput-root"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiFilledInput-input MuiInputBase-input css-10botns-MuiInputBase-input-MuiFilledInput-input"
                      id=":r2:"
                      placeholder=""
                      readonly=""
                      type="default"
                      value=""
                    />
                  </div>
                </div>
              </div>
              <span
                class="MuiTypography-root MuiTypography-bodyTinyMedium css-lj1nv7-MuiTypography-root"
              >
                (Optional)
              </span>
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiBox-root css-1iqw0sw"
              >
                <div
                  class="MuiBox-root css-132diu7"
                >
                  <span
                    style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
                  >
                    <span
                      style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                    >
                      <img
                        alt=""
                        aria-hidden="true"
                        src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2720%27%20height=%2720%27/%3e"
                        style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                      />
                    </span>
                    <img
                      alt=""
                      data-nimg="intrinsic"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </span>
                </div>
                <div
                  class="MuiFormControl-root MuiTextField-root custom-input css-11mrpan-MuiFormControl-root-MuiTextField-root"
                  withicon="true"
                >
                  <label
                    class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled MuiFormLabel-root MuiFormLabel-colorPrimary css-1suhfmo-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="false"
                    for=":r3:"
                    id=":r3:-label"
                  >
                    Insurance Carrier
                  </label>
                  <div
                    class="MuiFilledInput-root MuiFilledInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-qax0ed-MuiInputBase-root-MuiFilledInput-root"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiFilledInput-input MuiInputBase-input css-10botns-MuiInputBase-input-MuiFilledInput-input"
                      id=":r3:"
                      placeholder=""
                      readonly=""
                      type="default"
                      value=""
                    />
                  </div>
                </div>
              </div>
              <span
                class="MuiTypography-root MuiTypography-bodyTinyMedium css-lj1nv7-MuiTypography-root"
              >
                (Optional)
              </span>
            </div>
            <button
              class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root searchButton css-1unuvva-MuiButtonBase-root-MuiButton-root"
              data-testid="searchbtn"
              tabindex="0"
              type="submit"
            >
              Search
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </form>
        </div>
        <div />
      </div>
      <div
        class="MuiGrid-root MuiGrid-container css-u2wmcx-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1gntrgc-MuiGrid-root"
        >
          Filter Result content here
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1k4mpik-MuiGrid-root"
        >
          <span
            class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-18lrjg1-MuiCircularProgress-root"
            role="progressbar"
            style="width: 40px; height: 40px;"
          >
            <svg
              class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
              viewBox="22 22 44 44"
            >
              <circle
                class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
                cx="44"
                cy="44"
                fill="none"
                r="20.2"
                stroke-width="3.6"
              />
            </svg>
          </span>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
